tmail-backend/blob/blob-guice/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
for "guice" binding module
We should located it to [tmail-backend/guice](https://github.com/linagora/tmail-backend/tree/master/tmail-backend/guice) directory